documentation/notes/joss_paper/paper.md

```diff
@@ -47,11 +47,10 @@ A future development is planned that will enable the user to specify the basis o
 
 The model implementation consists of submodules to set up the model's parameters and to compute the tensor that defines the coefficients in the tendencies of the model variables; more details can be found in @DDV2016.
 This tensor is used by the code to compute the tendencies function and its Jacobian matrix. These functions can then be fed to the `qgs` built-in Runge-Kutta integrator or 
-to another integrator implemented by the user. As an example, the usage of the Julia DifferentialEquations.jl [@RN2017] integration package through the Python diffeqpy [@diffeqpy] package is provided.
-Technical details about this implementation can be found in the *Code Description* section of the included documentation.
+to another integrator implemented by the user. As an example, the usage of the Julia `DifferentialEquations.jl` [@RN2017] integration package through the Python `diffeqpy` [@diffeqpy] package is provided.
```

Comment:
Generally, there are a number of places where monospaced font would be useful as an immediate indicator that the text in question is naming a package, rather than standard text. It is up to you how far you wish to go with 'monospacing' packages mentioned in the text, but since you set "qgs" itself like that throughout the paper, for consistency I would recommend doing the same for other packages, notably those  listed out under the 'State of the Field' section. In (other) cases of NumPy and SciPy, I think plain text is fine, as they tend to be used as names, whereas if you had used the actual package names `numpy` or `scipy` those would be best as literals.

Reply:
You are right, thank you.